tests/unittests/execution_cache/execution_cache_io_tests.py

```
@@ -0,0 +1,92 @@
+Integration tests for execution_cache.
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import unittest
+import fiftyone as fo
+
+from fiftyone.operators.cache import execution_cache
+from fiftyone.operators.executor import ExecutionContext
+from fiftyone.operators.store import ExecutionStore
```

Ruff (0.8.2)

`fiftyone.operators.store.ExecutionStore` imported but unused

Remove unused import: `fiftyone.operators.store.ExecutionStore`

(F401)